src/agents/utility.jl

```diff
@@ -1070,6 +1070,10 @@ function solve_agent_problem!(
     VIUDER_Utility = get_new_jump_model(utility_opts.solvers)
     delta_t = get_delta_t(model_data)
 
+    # z_to_h_dict = get_one_to_many_dict(model_data.index_z_h_map, 1)
+    z_to_h_dict = get_one_to_many_dict(model_data.index_z_h_map, :index_z)
+    @show z_to_h_dict
```

Comment:
```suggestion
    z_to_h_dict = get_one_to_many_dict(model_data.index_z_h_map, :index_z)
```

Sorry--I think I left this debugging code in when I passed the branch back last time.